lib/pharos/phases/configure_cluster_info.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Pharos
+  module Phases
+    class ConfigureClusterInfo < Pharos::Phase
+      title "Configure cluster-info"
+
+      def client
+        @client ||= Pharos::Kube.client(@host.address)
```

Comment:
These should always be `@master.address`, unless it were really doing something specific to the kube API on that specific host.

Reply:
Fixed